sw/host/opentitanlib/src/io/gpio.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+
+/// A trait which represents a GPIO interface.
+pub trait Gpio {
+    /// Reads the value of the the GPIO pin `id`.
+    fn read(&self, id: usize) -> Result<bool>;
```

Comment:
This is kinda nitpicky, but is `usize` the right type for a GPIO pin ID? Pin IDs don't seem to have anything to do with pointers or memory allocation sizes, so I suspect a fixed-sized integer would be a better choice.

Reply:
Changed to u32.